dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/subcommands/impl/All.java

```diff
@@ -138,4 +154,21 @@ protected GitForkBranch getGitForkBranch(String image, String tag, Namespace ns)
     protected PullRequests getPullRequests(){
         return new PullRequests();
     }
+
+    protected RateLimiter getRateLimiter(Namespace ns){
+        long rateLimit = Constants.DEFAULT_RATE_LIMIT;
+        Duration rateLimitDuration = Constants.DEFAULT_RATE_LIMIT_DURATION;
+        Duration tokenAddingRate = Constants.DEFAULT_TOKEN_ADDING_RATE;
+        
+        if(ns.get(Constants.RATE_LIMIT)!=null){
+            rateLimit = ns.get(Constants.RATE_LIMIT);
+        }
+        if(ns.get(Constants.RATE_LIMIT_DURATION)!=null){
+            rateLimitDuration = Duration.parse(ns.get(Constants.RATE_LIMIT_DURATION));
+        }
+        if(ns.get(Constants.TOKEN_ADDING_RATE)!=null){
+            tokenAddingRate = Duration.parse(ns.get(Constants.TOKEN_ADDING_RATE));
+        }
+        return new RateLimiter(rateLimit, rateLimitDuration, tokenAddingRate);
+    }
```

Comment:
This looks like it was copied in at least two places?  Can't we can create a single factory class for creating rate limiters and just call it in both places?